lib/jsxgettext.js

```diff
@@ -197,37 +213,52 @@ function parse(sources, options) {
     }
 
      walk.simple(ast, {'CallExpression': function (node) {
-        var arg = getTranslatable(node, options);
-        if (!arg)
+        var args = getTranslatable(node, options);
+        if (!args)
           return;
-
+        var funcName = args.pop();
```

Comment:
I think this logic should be isolated into `getTranslatable`. It should only return a structure with the fields `context` and `translatable`.